src/rpcgovernance.cpp

```diff
@@ -284,6 +285,100 @@ UniValue gobject(const UniValue& params, bool fHelp)
         return returnObj;
     }
 
+    if(strCommand == "vote-many")
+    {
+        if(params.size() != 4)
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "Correct usage is 'gobject vote-many <governance-hash> [funding|valid|delete] [yes|no|abstain]'");
+
+        uint256 hash;
+        std::string strVote;
+
+        hash = ParseHashV(params[1], "Object hash");
+        std::string strVoteAction = params[2].get_str();
```

Comment:
I did not add terminology. These are the exact same variable names used in vote-alias and vote-conf:

https://github.com/dashpay/dash/blob/v0.12.1.x/src/rpcgovernance.cpp#L221-L223
https://github.com/dashpay/dash/blob/v0.12.1.x/src/rpcgovernance.cpp#L299-L301